ChatApp/package.json

```diff
@@ -10,14 +10,20 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@invertase/react-native-apple-authentication": "^2.2.2",
+    "@react-native-community/datetimepicker": "^6.5.4",
     "@react-native-firebase/app": "^14.11.1",
     "@react-native-firebase/auth": "^14.11.1",
-    "@react-native-firebase/firestore": "^14.11.1",
+    "@react-native-firebase/firestore": "14.12.0",
     "@react-native-firebase/storage": "^14.11.1",
+    "@react-native-google-signin/google-signin": "^8.1.0",
     "@react-navigation/native": "^6.0.13",
     "@react-navigation/native-stack": "^6.9.1",
     "react": "18.1.0",
     "react-native": "0.70.4",
+    "react-native-fbsdk-next": "^11.0.1",
+    "react-native-image-picker": "^4.10.0",
+    "react-native-modal": "^13.0.1",
```

Comment:
We can create a modal using the default Modal component from React Native.

Reply:
Done